### PR TITLE
Revamp frontend

### PR DESCRIPTION
**Pull request by JulienMirval**
- Clean up useless parts
- Simpler design
- Fix broken buttons due to recent changes
- Adaptive tree

### CODE REVIEW

File: src/components/Execution/index.jsx [hunk 1]
```diff
@@ -26,10 +22,31 @@ export const Execution = ({ nodes }) => {
   const [webhooks, setWebhooks] = useState([])
   const [singleNode, setSingleNode] = useState(data[0])
 
-  const createWebhooks = useCallback(async () => {
-    const query = async name => {
-      setIsWorking(true)
+  const fetchWebhooks = useCallback(async () => {
+    let { data: webhooks } = await client.collection(TRIGGERS_DOCTYPE).all()
```
Comment (paultranvan) [inline src/components/Execution/index.jsx:24]: We avoid using collection when we can, because it bypasses all the store features. Moreover, we have a `useQuery` hooks to do the querying work. And from what I see, we even need `useQueryAll`: https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useQueryAll.jsx

Also, I suggest you to group all mango queries into a separate `queries.js` file. We do that for every app, see [here](https://github.com/cozy/cozy-libs/blob/master/packages/cozy-mespapiers-lib/src/helpers/queries.js) for example.

So at the end, that would look like this: 
```
import { triggersQuery } from 'queries'
const { data: webhooks } = useQueryAll(
    triggersQuery.definition,
    triggersQuery.options
  )
```

And the triggersQuery would look like this: 
```
export const triggersQuery = () => ({
  definition: Q(TRIGGERS_DOCTYPE).partialIndex({ type: '@webhook' })
  options: {
    as: `${TRIGGERS_DOCTYPE}/type`,
    fetchPolicy: defaultFetchPolicy
  }
})
```
Note the partial index makes the filter below useless. The sort is also probably unecessary as the docs are already sorted by their id in couchdb

File: src/components/Execution/FullAggregation.jsx [hunk 2]
```diff
@@ -54,19 +58,19 @@ const FullAggregation = ({ nodes, webhooks }) => {
...
-      { ...nodes[7], level: 2, nbChild: 0, parents: aggregators },
-      { ...nodes[8], level: 2, nbChild: 0, parents: aggregators },
-      { ...nodes[9], level: 2, nbChild: 0, parents: aggregators }
+      { ...data[3], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[4], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[5], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[6], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[7], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[8], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[9], level: 2, nbChild: 0, parents: aggregators }
```
Comment (paultranvan) [inline src/components/Execution/FullAggregation.jsx]: This looks very static, I suppose this won't work with a dynamic tree?

File: src/lib/createTreeExported.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
+/**
+ * FIXME: THIS FILE IS A WORKAROUND TO CONFLICTING IMPORT STYLES.
```
Comment (paultranvan) [inline src/lib/createTreeExported.js:2]: Could you elaborate a bit here, and say the file is duplicated because it is needed in both node.js (common.js) and web (ESM) environment?

File: src/components/Execution/FullAggregation.jsx (hunk 2 above)
Comment (JulienMirval) [reply src/components/Execution/FullAggregation.jsx]: Removed in later commits

File: src/components/Execution/SingleNodeAggregation.jsx [hunk 4]
```diff
@@ -49,43 +55,72 @@ const SingleNodeAggregation = ({ node }) => {
...
+        >
+          Launch execution
+        </Button>
+      </div>
     </div>
   )
 }
 
-export default SingleNodeAggregation
+export default queryConnect({
```
Comment (paultranvan) [inline src/components/Execution/SingleNodeAggregation.jsx]: I initially missed that, but it shouldn't be necessary: why did you add it?

File: src/components/Execution/SingleNodeAggregation.jsx [hunk 5]
```diff
@@ -1,25 +1,35 @@
...
+  const query = nodesQuery()
+  const { fetch, isLoading } = useQuery(query.definition, query.options)
+  const [nodes, setNodes] = useState()
   const [nbShares, setNbShares] = useState(3)
   const [pretrained, setPretrained] = useState(true)
-  const [node, setSingleNode] = useState(data[0])
+  const [node, setSingleNode] = useState()
+  const [isWorking, setIsWorking] = useState(false)
+
+  useEffect(() => {
```
Comment (paultranvan) [inline src/components/Execution/SingleNodeAggregation.jsx:24]: As usual in those very non satisfying cases: could you add a FIXME comment saying that it shouln't be necessary if `useQuery` was properly working?

File: src/components/Execution/index.jsx [hunk 6]
```diff
@@ -13,70 +11,57 @@ import {
...
-    let { data: webhooks } = await client.collection(TRIGGERS_DOCTYPE).all()
-
-    setWebhooks(
-      webhooks
-        .filter(hook => hook.type === '@webhook')
-        .sort((a, b) => (a.id > b.id ? 1 : -1))
-    )
-  }, [client, setWebhooks])
 
   useEffect(() => {
```
Comment (paultranvan) [inline src/components/Execution/index.jsx:23]: Same remark about FIXME comment

File: src/components/Operations/index.jsx [hunk 7]
```diff
@@ -1,35 +1,42 @@
...
+import { useEffect } from 'react'
+
+export const Operations = () => {
+  const query = bankQuery()
+  const { isLoading, fetch } = useQuery(query.definition, query.options)
+  const [banks, setBanks] = useState()
 
-export const Operations = ({ bank }) => {
-  const { isLoading, data } = bank
+  useEffect(() => {
```
Comment (paultranvan) [inline src/components/Operations/index.jsx:17]: Same remark about FIXME

**Review by paultranvan — APPROVED** [review]

File: src/components/Execution/SingleNodeAggregation.jsx (hunk 4 above)
Comment (JulienMirval) [reply src/components/Execution/SingleNodeAggregation.jsx]: It's gone in the recent commits
